whitesands/code/modules/research/techweb/all_nodes.dm

```diff
@@ -13,7 +13,7 @@
 	display_name = "Basic Shuttle Research"
 	description = "Research the technology required to create and use basic shuttles."
 	prereq_ids = list("bluespace_travel", "adv_engi")
-	design_ids = list("engine_plasma", "engine_ion", "engine_heater", "engine_smes", "shuttle_helm") //, "rapid_shuttle_designator")
+	design_ids = list("engine_plasma", "engine_ion", "engine_heater", "engine_smes", "shuttle_helm", "shuttle_autopilot") //, "rapid_shuttle_designator")
```

Comment:
hey while you're here why don't you just move all of the `shuttle_tech` research nodes to voidcrew